lib/ramfs/ramfs_vnops.c

```diff
@@ -55,6 +55,12 @@
 #include <fcntl.h>
 #include <vfscore/fs.h>
 
+/* 16 bits are enough for file mode as defined by POSIX, rest we can use */
```

Comment:
It not very clear to me at what  "rest we can use". Could you be more explicit there, please?

Reply:
I'll add a "the" for grammar pedantry.